commit 34f0c83dc026f5403fd0e408a31efc56208bd47b
Author: Khem Raj <raj.khem@gmail.com>
Date:   Thu Jul 21 21:26:45 2016 -0700

    trace_shed: Drop useless and wrong WEXITSTATUS(status)
    
    The thread return status is either -1 or pointer to the table. So there
    is no point in passing it to WEXITSTATUS() and priting the value.
    
    The code started to fail to compile after glibc commit:
    https://sourceware.org/git/?p=glibc.git;a=commitdiff;h=b49ab5f4503f36dcbf43f821f817da66b2931fe6;hp=5f5682b9654101ccaf375c2814cbddbe6033a725
    
    trace_sched.c:425:16: error: invalid operands to binary & (have
    'thread_sched_t * {aka struct <anonymous> *}' and 'int')
          thrd_ndx, WEXITSTATUS(status));
    
    Signed-off-by: Khem Raj <raj.khem@gmail.com>
    Acked-by: Cyril Hrubis <chrubis@suse.cz>

diff --git a/testcases/kernel/sched/tool/trace_sched.c b/testcases/kernel/sched/tool/trace_sched.c
index 781568613138..71caf239a825 100644
--- a/testcases/kernel/sched/tool/trace_sched.c
+++ b/testcases/kernel/sched/tool/trace_sched.c
@@ -421,8 +421,8 @@ int main(int argc,		/* number of input parameters.                        */
 		} else {
 			if (status == (thread_sched_t *) - 1) {
 				fprintf(stderr,
-					"thread [%d] - process exited with errors %d\n",
-					thrd_ndx, WEXITSTATUS(status));
+					"thread [%d] - process exited with exit code -1\n",
+					thrd_ndx);
 				exit(-1);
 			} else {
 				exp_prio[thrd_ndx] = status->exp_prio;
